### ISSUE TITLE
Create new mixin and new login and signup pages to support #551

### ISSUE THREAD

**Issue by luchia** [TeamMentor/TM_4_0_Design#669]
As per discussion on #663

**Comment by roman87** [comment]
@luchia, actually there is no need to create a new page, there is a need to create a mixin so that we can safely display an email address without un-escaping to HTML. Like the comment says here:
https://github.com/TeamMentor/TM_4_0_Design/pull/663#discussion_r27800614

Also I am bumping it up since it is pretty important and @michaelhidalgo is waiting on this for #551

**Comment by michaelhidalgo** [comment]
Actually, @DinisCruz  please correct me if I'm wrong, but we will need two jade pages for the Login and the Signup since both pages will need to load the new mixin

At least that is what we have done with other cases : 

![screen shot 2015-04-07 at 12 54 03 pm](https://cloud.githubusercontent.com/assets/1732035/7030863/e417e82e-dd24-11e4-85a6-3d5978629637.png)

**Comment by DinisCruz** [comment]
well if we create a generic 'got an issue' page then we would only need one.

The question will then be on what scenarios to redirect the user to it

**Comment by luchia** [comment]
Notes from our meeting this morning -

If the user is logging in and for whatever reason the application can't talk to the 3.5 backend, we need to display an error that says 'Sorry TM is unavailable, please contact us at email.' 
It should look like an alert, not like one our existing error pages.

**Comment by luchia** [comment]
So, I just created a new branch for this which I pushed. Assigning to @michaelhidalgo to be merged into his #551 branch.

**Comment by luchia** [comment]
Here's the [branch](https://github.com/TeamMentor/TM_4_0_Design/tree/Issue_669) @michaelhidalgo.

**Comment by michaelhidalgo** [comment]
Hi @luchia , I think here https://github.com/TeamMentor/TM_4_0_Design/blob/Issue_669/source/jade/_mixins/user-mixins.jade#L124 you need to append the mailto html tag so it looks like

```jade
#message= 'Error: ' + viewModel.errorMessage
+tm-support-email
```

**Comment by luchia** [comment]
Just pushed it @michaelhidalgo.

**Comment by roman87** [comment]
@michaelhidalgo Don't forget to assign this to appropriate person :)

**Comment by roman87** [comment]
Ok nevermind, somehow my page didn't update

**Comment by michaelhidalgo** [comment]
:)

**Comment by michaelhidalgo** [comment]
Hi @luchia it seems like the font size of the email is a bigger than the actual text from the alert.

Do you think we can make them same size?

![screen shot 2015-04-08 at 4 04 08 pm](https://cloud.githubusercontent.com/assets/1732035/7056495/78ffa168-de09-11e4-8108-451838772fe0.png)

**Comment by michaelhidalgo** [comment]
This has been done. The mixin and the page were created.

I'm going to close this issue since we are tracking it here https://github.com/TeamMentor/TM_4_0_Design/pull/663
